exercises/bank-account/src/test/java/BankAccountTest.java

```diff
@@ -0,0 +1,201 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.Random;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class BankAccountTest {
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+    private BankAccount bankAccount;
+
+    @Before
+    public void setup() {
+        bankAccount = new BankAccount();
+    }
+
+    @Test
+    public void newlyOpenedAccountHasEmptyBalance() throws BankAccountActionInvalidException {
+        bankAccount.open();
+
+        assertEquals(0, bankAccount.getBalance());
+    }
+
+    @Ignore
```

Comment:
All `@Ignore` annotations should include the standard hint: `"Remove to run test"` :)